cfg/atari2600.cfg

```diff
@@ -0,0 +1,22 @@
+# Atari VCS 2600 linker configuration file for cc65
+#
+# Florent Flament (contact@florentflament.com), 2017
+
+SYMBOLS {
+    __STACKSIZE__: type = weak, value = $0010;   # 16 Bytes system stack
+}
+
+MEMORY {
+    RAM: file = "", start = $0080, size = $0080 - __STACKSIZE__, define = yes;
+    ROM: file = %O, start = $F000, size = $1000;
```

Comment:
I think that the "`, fillval = $FF`" attribute needs to be in that line (unused locations in PROMs should have their bits set to one).